src/sagemaker_xgboost_container/algorithm_mode/hyperparameter_validation.py

```diff
@@ -162,12 +163,12 @@ def interaction_constraints_validator(value, dependencies):
                                      required=False, tunable=True,
                                      tunable_recommended_range=hpv.Interval(min_closed=0.1, max_closed=1,
                                                                             scale=hpv.Interval.LINEAR_SCALE)),
-        hpv.ContinuousHyperparameter(name="lambda", range=hpv.Interval(min_closed=0), required=False,
-                                     tunable=True,
+        hpv.ContinuousHyperparameter(name="lambda", range=hpv.Interval(min_closed=0),
+                                     required=False, tunable=True,
                                      tunable_recommended_range=hpv.Interval(min_closed=0, max_closed=1000,
                                                                             scale=hpv.Interval.LINEAR_SCALE)),
-        hpv.ContinuousHyperparameter(name="alpha", range=hpv.Interval(min_closed=0), required=False,
-                                     tunable=True,
+        hpv.ContinuousHyperparameter(name="alpha", range=hpv.Interval(min_closed=0),
+                                     required=False, tunable=True,
```

Comment:
are these changes still necessary? I think moving `required` to newline is not needed anymore since you are not using the `alias` parameter.